packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -449,7 +452,8 @@ class InkResponse extends StatefulWidget {
   }
 
   @override
-  _InkResponseState<InkResponse> createState() => _InkResponseState<InkResponse>();
+  _InkResponseState<InkResponse> createState() =>
```

Comment:
Actually please keep these statements in a single line if that is clearer. Flutter prefers an 80-column line but doesn't enforce so, if breaking the line makes it less clear. There are many more cases later.